server/models/schemas/user.js

```diff
@@ -110,8 +110,20 @@ const schema = new Schema({
             objectsScaling: { type: Types.String, required: false, enum: ['default', 'clamped'], default: 'default' },
             objectsMinimumScale: { type: Types.Number, required: false, default: 8 },
             objectsMaximumScale: { type: Types.Number, required: false, default: 16 },
-            nebulaDensity: { type: Types.String, required: false, enum: ['none', 'sparse', 'standard', 'abundant'], default: 'standard' },
             antiAliasing: { type: Types.String, required: false, enum: ['enabled', 'disabled'], default: 'enabled' },
+            background:{
+              nebulaFrequency: { type: Types.Number, required: false, default: 12 },
+              nebulaDensity: { type: Types.Number, required: false, default: 3 },
+              nebulaOpacity: { type: Types.Number, required: false, default: 1.0 },
+              moveNebulas: { type: Types.String, required: false, enum: ['enabled', 'disabled'], default: 'enabled' },
+              nebulaMovementSpeed: { type: Types.Number, required: false, default: 1.0 },
+              starsOpacity: { type: Types.Number, required: false, default: 1.0 },
+              blendMode: { type: Types.String, required: false, enum: ['ADD', 'NORMAL'], default: 'NORMAL' },
+              backgroundStars: { type: Types.String, required: false, enum: ['enabled', 'disabled'], default: 'enabled' },
+              nebulaColor1: { type: Types.String, required: false, default: '#ffad42' },
+              nebulaColor2: { type: Types.String, required: false, default: '#ffad42' },
+              nebulaColor3: { type: Types.String, required: false, default: '#ff3b86' }
```

Comment:
`Color` vs. `Colour`, keep it consistent to whatever already exists in the code! This includes labels.

Reply:
ok, i'll refactor it